backend/controllers/prisonerProfile/prisonerVisits.ts

```diff
@@ -8,7 +8,26 @@ export const VISIT_TYPES = [
   { value: 'OFFI', text: 'Official' },
 ]
 
-const calculateStatus = ({ cancelReasonDescription, completionStatusDescription, completionStatus, startTime }) => {
+const calculateDateAndStatusFilter = (status: string, fromDate: string, toDate: string) => {
+  const fromAsDate = fromDate ? moment(fromDate, 'DD/MM/YYYY') : undefined
+  const toAsDate = toDate ? moment(toDate, 'DD/MM/YYYY') : undefined
+
+  const now = moment().startOf('day')
+  if (status === 'EXP') return { visitStatus: 'SCH', fromAsDate, toAsDate: toAsDate?.isBefore(now) ? toAsDate : now }
+  if (status === 'SCH') return { visitStatus: 'SCH', fromAsDate: fromAsDate?.isAfter(now) ? fromAsDate : now, toAsDate }
```

Comment:
Similarly front end wants a `Scheduled` status to indicate visits in the past, so need to play around with the from and to dates to Prison API to cope